apiserver/params/internal.go

```diff
@@ -950,6 +950,7 @@ type LogMessage struct {
 	Module    string    `json:"mod"`
 	Location  string    `json:"loc"`
 	Message   string    `json:"msg"`
+	Labels    []string  `json:"lab"`
```

Comment:
Really minor nit; maybe it would read better like `msg`; feel free to ignore
 
```suggestion
	Labels    []string  `json:"lbl"`
```